src/lib/converters/files.converter.ts

```diff
@@ -182,9 +199,16 @@ const generateCompleteIconSet = async (
   svgDefinitions: SvgDefinition[],
   outputDirectory: string,
   iconsFolderName: string,
-  completeIconSetName: string
+  completeIconSetName: string,
+  interfaceName?: string,
+  modelFileName?: string
```

Comment:
Seems like the`modelFileName` is never passed through, therefore it will always be undefined. I think the `modelFileName` should also be passed through in the calls of this method.